lib/ruby_lsp/setup_bundler.rb

```diff
@@ -255,7 +255,9 @@ def bundler_settings_as_env
       # "vendor/bundle"`
       settings.all.to_h do |e|
         key = Bundler::Settings.key_for(e)
-        [key, settings[e].to_s]
+        value = Array(settings[e]).join(":")
```

Comment:
Does Bundler always use `:` as the separator? Or would it use `;` on Windows?

Reply:
Oh, since they have helper methods for those, we should probably just use them directly instead of re-implementing it on our side.

Can we change it to use their API?

```ruby
value = settings.array_to_s(settings[e]).to_s
```

Reply:
Unfortunately, `array_to_s` is private 😞